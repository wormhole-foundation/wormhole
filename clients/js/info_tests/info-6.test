info wrapped ethereum 0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48 sui
