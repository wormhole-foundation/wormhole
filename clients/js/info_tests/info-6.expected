0x5d4b302506645c37ff133b98c4b50a5ae14841659738d6d733d59d0d217a93bf::coin::COIN
